src/main/java/utils/Utils.java

```diff
@@ -111,6 +111,11 @@ public static void chronologicalOrder(List<RefactoringInfo> infos) {
    */
   public static int skipJavadoc(String text, int line, boolean skipAnnotations) {
     String[] lines = text.split("\r\n|\r|\n");
+    if (line >= lines.length) {
+      System.out.println(text);
```

Comment:
I would like to recommend using log4j lib for logging instead of System.out.println. We could leave it the way it is, but it would be great to fix it in the next PR.

Reply:
It was used for debugging since this case should not happen, but I agree!